assets/js/breadcrumbs/mobile-breadcrumb.js

```diff
@@ -0,0 +1,139 @@
+var config = {
```

Comment:
I ran the linter manually a couple of times to be sure, and it didn't throw any errors. Will be sure to hook up the pre-commit hooks so all future runs lint without manual intervention.

Reply:
Per @jbalboni request, I inlined this script using UglifyJS. I'll remove it here if the inlined version passes muster.

Reply:
Could we have the breadcrumb transformer run uglify on this script and inline the result? Having the uglified version as the only source is going to be hard to maintain.

Reply:
Also, I believe the linter ignores files in this directory, since these files aren't run through babel and can't use ES6.

We may want to change this process.